script.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+	//DO NOT CHANGE CODE BELOW
+        var num1 = parseInt($('#num1').val());
+        var num2 = parseInt($('#num2').val());
+        clear();
+        addNumbers(num1, num2);
+        var result = multiplyNumbers(num1, num2);
+        $('#result-mult').text(result);
+	//DO NOT CHANGE CODE ABOVE
+
+    });
+
+	/*
+	 Below this comment, create a function 
+	 named addNumbers, which accepts two parameters.
+	 The function should add the two parameters together
+	 and write the result to the element with the id
+	 result-add
+    */
+    function addNumbers(num1,num2){
+        document.getElementById("num1","num2").innerHTML=addNumbers;
+        $("#result-add").append('<div></div>');
+        var(result-add) = num1 + num2;
+        return num1 + num2;
+    }
+
+	/*
+	 Below this comment, create a function
+	 named multiplyNumbers, which accepts two parameters.
+	 The function should multiply the two parameters together
+	 and return the result.
+    */
+    function multiplyNumbers(num1,num2){
+        document.getElementbById("num1","num2").innerHTML=multiplyNumbers;
+        $("#result-mult").append('<div></div>');
+        var (result-mult) = num1 * num2;
+        return num1 * num2;
```

Comment:
actually - with line 38 written this way - that is all you need in this function. Line 37 isn't doing anything.